lblod/app-digitaal-loket
# Update name & concept schemes

Test:

- Use updated enrich submission service (https://github.com/lblod/enrich-submission-service/pull/13)
- login as gemeente deinze
- start new toezicht form 
- choose any dossier that has 'Betreffend bestuur van de eredienst erediens' field (eg: Besluit over meerjarenplan(aanpassing) eredienstbestuur)
- check if dropdown items are scoped to related erediensten of that gemeente
